### PR TITLE
Fix #336: provided a clear description of where to add the repo name in the constants.js file.

### LINKED ISSUE #336: Update the readme me file
**Is your feature request related to a problem? Please describe.**
Readme file's [this heading](https://github.com/oppia/oppiabot?tab=readme-ov-file#installing-the-bot-on-a-repository) doesn't provide a clear description of where to add the repo name in the ``constant.js`` file.

**Describe the solution you'd like**
Add the description

**Describe alternatives you've considered**

**Additional context**

### PR DESCRIPTION
**Pull request by Flying-Pegasus**
## Explanation
Updated the 'Installing the Bot on a Repository' section,
Added the bulletpoints to show step by step process.

The new lines I have added :
3. Added the repository name in the **`constants.js` file**:  
   - Open the **`constants.js`** file in the bot's source code.  
   - Navigate to the `checksWhitelist` object.  
   - Add a new entry with your repository name (in lowercase) as the key. The value should define the checks and events specific to your repository.  

   **Example:**
   If your repository name is `my-repo`, the `checksWhitelist` object will look like this:  
   ```javascript
   const checksWhitelist = {
     // Existing entries...
     'my-repo': {
       [openEvent]: [claCheck],
       [reopenEvent]: [],
       [openEventGithubActions]: [claCheckGithubAction],
       [reopenEventGithubActions]: [claCheckGithubAction],
       [synchronizeEvent]: [mergeConflictCheck],
       [closeEvent]: [allMergeConflictCheck],
       [editEvent]: [],
       [issuesLabelEvent]: [],
       [issuesAssignedEvent]: [],
       [pushEvent]: [],
       [pullRequestReviewEvent]: [pullRequestReviewCheck],
       [issueCommentCreatedEvent]: [respondToReviewCheck],
     },
   };
```
   
4.Save the changes to **`constants.js`**.

## Checklist
- [x] I have successfully deployed my own instance of Oppiabot.
  - You can find instructions for doing this [here](https://github.com/oppia/oppiabot/wiki/Deploying-your-own-instance-of-the-oppiabot).
- [x] I have manually tested all the changes made in this PR following the [manual tests matrix](https://github.com/oppia/oppiabot/wiki/Manual-Tests-Matrix).

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -79,7 +79,38 @@ from .env file, Otherwise app will not work locally.
 
 ## Installing the bot on a repository
 
-You'll need to identify a target repository and install the bot by clicking the **Install** button on the settings page of your app, e.g `https://github.com/settings/apps/my-oppiabot-testing-app/installations`. In the `.env` file put your github account name in `WHITELISTED_ACCOUNTS` and also add your repository (name in small caps) in the `constants.js` file locally.
+
+1. Identify a target repository and install the bot by clicking the Install button on the settings page of your app, e.g., **`https://github.com/settings/apps/my-oppiabot-testing-app/installations`** 
+2. Update the **`.env` file**:
+   - Add your GitHub account name to `WHITELISTED_ACCOUNTS`.  
+3. Add the repository name in the **`constants.js` file**:
```
Comment (seanlip) [inline README.md:86]: Please add a blank line above "3." and above "2.", similar to how you have a space between sections 3. and 4.

**Review by seanlip — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (Flying-Pegasus) [reply README.md:88]: Screenshot 2024-12-10 185802

@seanlip will this work?

**Review by seanlip — APPROVED** [review]
Looks good. Thanks!

**Comment by seanlip** [comment]
(And congrats on your first PR to Oppia!)

**Comment by Flying-Pegasus** [comment]
Thank you @seanlip for guiding me. Looking forward to solve more issues.

**Pull request merged**